test/image/mocks/scatter_fill_corner_cases.json

```diff
@@ -75,7 +75,6 @@
     {
       "x": [1.5],
       "y": [1.25],
-      "fill": "tonexty",
```

Comment:
Oh this change I think is actually required due to a change in the stacked area commit https://github.com/plotly/plotly.js/pull/2960/commits/be38e932ec8e0ac350a6f21d2d8e6fabf4f8cfed#diff-33c02cd37e7a4c951059a3c93221ac4eR175 - we were accidentally treating a length-1 trace as filling to itself (since its start and end points are the same!) but we shouldn't do that... therefore this trace, since it's the first on its subplot, should interpret `'tonexty'` as `'tozeroy'`.